sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -9633,6 +10251,112 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
+        Azure.ResourceManager.Sql.Models.NSPConfigPerimeter System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigPerimeter>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigPerimeter>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigPerimeter>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class NSPConfigProfile : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Sql.Models.NSPConfigProfile>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigProfile>
+    {
+        public NSPConfigProfile() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.Sql.Models.NSPConfigAccessRule> AccessRules { get { throw null; } }
+        public string AccessRulesVersion { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Sql.Models.NSPConfigProfile System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Sql.Models.NSPConfigProfile>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Sql.Models.NSPConfigProfile>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Sql.Models.NSPConfigProfile System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigProfile>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigProfile>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigProfile>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class NSPProvisioningIssue : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Sql.Models.NSPProvisioningIssue>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPProvisioningIssue>
```

Comment:
`NSP` is short for `NetworkSecurityPerimeter`, consider expanding it to the full name here.
And prepend `Sql` or others to avoid name dup across RP.

Reply:
Renamed to SqlNetworkSecurityPerimeterProvisioningIssue